src/Lykke.Service.PayInternal.Core/Domain/Transfer/ITransactionRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Lykke.Service.PayInternal.Core.Domain.Transfer
+{
+    public interface ITransactionRequest
+    {
+
+        string TransactionHash { get; set; }
+        List<IAddressAmount> SourceAmounts { get; set; }
+        string DestinationAddress { get; set; }
+        int CountConfirm { get; set; }
```

Comment:
What does this property supposed to mean?

Reply:
It's legacy part. This property is used while creating a new transfer request - is't usually set to 1. On my opinion, we do need to set it to 0 (for we do not have any transaction confirmations just after creating a transaction), but I'm not sure. Left unchanged.